opencensus/plugins/grpc/internal/filter.cc

```diff
@@ -17,6 +17,14 @@
 
 namespace opencensus {
 
+void CensusContext::RecordMessageEvent(bool sent, uint32_t id, uint32_t size) {
```

Comment:
Not a fan of the bool arg. IMO either expose span_ or make two helper functions in CensusContext. :)

Reply:
Split into 2 functions now.